update tiup doc (#15374)
pingcap/docs: tiup/tiup-cluster-topology-reference.md

- [global](#global): The cluster's global configuration. Some of the configuration items use the default values and you can configure them separately in each instance.
- [monitored](#monitored): Configuration for monitoring services, namely, the blackbox_exporter and the `node_exporter`. On each machine, a `node_exporter` and a `blackbox_exporter` are deployed.
- [server_configs](#server_configs): Components' global configuration. You can configure each component separately. If an instance has a configuration item with the same name, the instance's configuration item will take effect.

> - [component_versions](/tiup/tiup-cluster-topology-reference.md#component_versions): Component version. You can configure it when a component does not use the cluster version. This section is introduced in tiup-cluster v1.14.0.

Reviewer: ```suggestion
- [component_versions](#component_versions): Component version. You can configure it when a component does not use the cluster version. This section is introduced in tiup-cluster v1.14.0.
```

- `enable_tls`: Specifies whether to enable TLS for the cluster. After TLS is enabled, the generated TLS certificate must be used for connections between components or between the client and the component. The default value is `false`.

> - `listen_host`: Default listen host。If it is empty, each instance will automatically set it to `::` or `0.0.0.0` by judging whether its own `Host` field contains `:`.Introduced in tiup-cluster v1.14.0

Reviewer: ```suggestion
- `listen_host`: Specifies the default listen host. If it is empty, each instance will automatically set it to `::` or `0.0.0.0` by judging whether its own `Host` field contains `:`. This field is introduced in tiup-cluster v1.14.0. 
```

> - `listen_host`: Specifies the default listen host. If it is empty, each instance will automatically set it to `::` or `0.0.0.0` by judging whether its own `Host` field contains `:`. This field is introduced in tiup-cluster v1.14.0. 

Reviewer: ```suggestion
- `listen_host`: Specifies the default listen IP address. If it is empty, each instance will automatically set it to `::` or `0.0.0.0` by judging whether its own `host` field contains `:`. This field is introduced in tiup-cluster v1.14.0. 
```

The above configuration specifies the global configuration of TiDB and TiKV.

### `component_versions`

> **Note:**
>

> > For components such as TiDB, TiKV, PD, TiCDC that share the same version number, there is no complete test to ensure that they can work properly in a cross-version mixing scenario. Therefore, it is recommended to only use such scenarios for testing purposes or operate under the help of support.

Reviewer: ```suggestion
> For TiDB, TiKV, PD, TiCDC, and other components that share their own version numbers, there are no complete tests to ensure that they work properly in a mixed-version scenario. Ensure that you use this section only for testing scenarios, or with the help of technical support.
```

`component_versions` is used to specify the version number of a certain component. When `component_versions` is not configured, each component either uses the same version number as the TiDB cluster (such as PD and TiKV), or uses the latest version (such as Alertmanager). When this field is configured, the corresponding component will use the specified version, and this version will be used in subsequent cluster expansion and upgrade operations. Users should only configure this when they need to fix the version number of a component. The main fields include:

- `tikv`: The version of the TiKV component
- `tiflash`: The version of the TiFlash component
- `pd`: The version of the PD component
- `tidb_dashboard`: The version of the standalone TiDB Dashboard component
- `pump`: The version of the Pump component
- `drainer`: The version of the Drainer component
- `cdc`: The version of the CDC component
- `kvcdc`: The version of the TiKV-CDC component
- `tiproxy`: The version of the TiProxy component
- `prometheus`: The version of the Prometheus component
- `grafana`: The version of the Grafana component
- `alertmanager`: The version of the Alertmanager component

> Example configuration for `component_versions`:

Reviewer: ```suggestion
The following is an example configuration for `component_versions`:
```

The above configuration specifies the global configuration of TiDB and TiKV.

### `component_versions`

> **Note:**
>

> > For TiDB, TiKV, PD, TiCDC, and other components that share their own version numbers, there are no complete tests to ensure that they work properly in a mixed-version scenario. Ensure that you use this section only for testing scenarios, or with the help of technical support.

Reviewer: ```suggestion
> For TiDB, TiKV, PD, TiCDC, and other components that share their own version numbers, there are no complete tests to ensure that they work properly in a mixed-version scenario. Ensure that you use this section only for testing scenarios, or with the help of [technical support](/support.md).
```

`component_versions` is used to specify the version number of a certain component. 

- When `component_versions` is not configured, each component either uses the same version number as the TiDB cluster (such as PD and TiKV), or uses the latest version (such as Alertmanager). 

> - When `component_versions` is configured, the corresponding component will use the specified version, and this version will be used in subsequent cluster expansion and upgrade operations. 

Reviewer: ```suggestion
- When `component_versions` is configured, the corresponding component will use the specified version, and this version will be used in subsequent cluster scaling and upgrade operations. 
```

> Make sure you only configure it when you need to fix the version number of a component. 

Reviewer: ```suggestion
Make sure you only configure it when you need to use a specific version of a component. 
```

> The main fields include:

Reviewer: ```suggestion
The main fields of `component_versions` include:
```
